allenact_plugins/manipulathor_plugin/manipulathor_constants.py

```diff
@@ -0,0 +1,116 @@
+"""Constant values and hyperparameters that are used by the environment."""
+import ai2thor
+import ai2thor.fifo_server
+from allenact_plugins.ithor_plugin.ithor_environment import IThorEnvironment
+
+MOVE_THR = 0.01
+ARM_MIN_HEIGHT = 0.450998873
+ARM_MAX_HEIGHT = 1.8009994
+MOVE_ARM_CONSTANT = 0.05
+MOVE_ARM_HEIGHT_CONSTANT = MOVE_ARM_CONSTANT
+
+ADITIONAL_ARM_ARGS = {
```

Comment:
I believe the the same comment as above applies to many of the constants below (e.g. if I make a new manipulathor task then I might choose a different set of `TRAIN_OBJECTS`). It might make sense to create a `armpointnav_constants` file and add them there instead.

Reply:
great idea. I did that and added it under manipulathor_plugin. But it kinda feels like to be in the wrong place. Should I move it to projects/manipulathor_baselines/armpointnav_baselines? What do you think?

Reply:
I like that idea :)!

Reply:
Didn't realize you'd already moved these, they look good at `allenact_plugins/manipulathor_plugin/armpointnav_constants.py`